src/components/shared/form-components/drop-down/list/list.jsx

```diff
@@ -102,7 +101,21 @@ export default function DropDownMenu({
               onKeyDown={e => handleKeyPress(e, option)}
               onClick={() => setValueAndCloseDropDown(option)}
             >
-              {option}
+              {pathname === "/paintings" ? (
```

Comment:
since were almost done i dont mind it, but i would definitley learn to avoid creating context specific logic inside a re-useable component. There is a way you can use props to create different types of functionality. Look at the pimary button as an example, if you pass an href it returns a link, if you don't pass an href it returns a button with an onclick handler.